test/workbox-core/node/controllers/test-WorkboxCore.mjs

```diff
@@ -1,60 +1,40 @@
 import {expect} from 'chai';
-import clearRequire from 'clear-require';
-import makeServiceWorkerEnv from 'service-worker-mock';
-
 import expectError from '../../../../infra/utils/expectError';
 import generateVariantTests from '../../../../infra/utils/generate-variant-tests';
+import core from '../../../../packages/workbox-core/index.mjs';
+import * as coreModule from '../../../../packages/workbox-core/index.mjs';
 
 describe(`workbox-core WorkboxCore`, function() {
-  before(function() {
-    const swEnv = makeServiceWorkerEnv();
-    Object.assign(global, swEnv);
-  });
-
-  beforeEach(function() {
-    clearRequire.all();
-    process.env.NODE_ENV = 'dev';
-  });
-
   describe(`core.assert.*`, function() {
-    it(`should expose assert in dev build`, async function() {
-      const coreModule = await import('../../../../packages/workbox-core/index.mjs');
-      const core = coreModule.default;
+    it(`should expose assert in dev`, async function() {
+      if (process.env.NODE_ENV == 'production') return this.skip();
 
       expect(core.assert).to.exist;
     });
 
     it(`should NOT expose assert in prod build`, async function() {
-      process.env.NODE_ENV = 'production';
-
-      const coreModule = await import('../../../../packages/workbox-core/index.mjs');
-      const core = coreModule.default;
+      if (process.env.NODE_ENV != 'production') return this.skip();
 
       expect(core.assert).to.not.exist;
     });
   });
 
   describe(`core.logLevel (getter)`, function() {
-    it(`should initialise to 'verbose' log level for dev build`, async function() {
-      const coreModule = await import('../../../../packages/workbox-core/index.mjs');
-      const core = coreModule.default;
+    it(`should initialise to 'verbose' log level in dev`, function() {
+      if (process.env.NODE_ENV == 'production') return this.skip();
+
       expect(core.logLevel).to.equal(coreModule.LOG_LEVELS.verbose);
     });
 
-    it(`should initialise to 'warn' log level for production build`, async function() {
-      process.env.NODE_ENV = 'production';
+    it(`should initialise to 'warn' log level in prod`, function() {
+      if (process.env.NODE_ENV != 'production') return this.skip();
```

Comment:
Nit: Throughout I'd use !== and ===

Reply:
Happy to change for consistency. BTW, Google style doesn't have an opinion on this (though closure style does and suggests `==` since with closure compiler you always know the type you're comparing).